lua/octo/completion.lua

```diff
@@ -48,38 +49,51 @@ local function get_repo_issues(repo, params)
 end
 
 function M.issue_complete(findstart, base)
-  -- :he complete-functions
+  -- the complete-functions
   if findstart == 1 then
     -- findstart
     local line = api.nvim_get_current_line()
     local pos = vim.fn.col(".")
-    local i, j = 0
+    local i, j = 0, 0
     while true do
       i, j = string.find(line, "#(%d*)", i + 1)
       if i == nil then
-        break
+        i, j = 0, 0
+        i, j = string.find(line, "@(%w*)", i + 1)
+        if i == nil then
+          break
+        end
       end
       if pos > i and pos <= j + 1 then
-        return i
+        -- I think subtracting 1 is necessary to include the first character
+        -- since lua is 1 indexed
+        return i - 1
       end
     end
     return -2
   elseif findstart == 0 then
     local repo = api.nvim_buf_get_var(0, "repo")
     local issues = get_repo_issues(repo)
```

Comment:
move this line down to the `if vim.startswith(base, "#")` block to avoid loading issues for `@` completion

Reply:
This should be resolved [here](https://github.com/pwntester/octo.nvim/pull/48/commits/a3dba031fc917d4318d87fd27a77ad5a94b31fbf)